src/adera/Shaders/Phong.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <vector>
+#include <Magnum/Shaders/Phong.h>
+#include <Magnum/GL/Mesh.h>
+#include <Magnum/GL/Texture.h>
+#include <Magnum/Math/Color.h>
+#include "osp/Active/activetypes.h"
+#include "osp/Active/ActiveScene.h"
+#include "osp/Resource/Resource.h"
+
+namespace adera::shader
+{
+
+
+class Phong : public Magnum::Shaders::Phong
```

Comment:
If you're only going to have public member functions, maybe just make it a struct?

Reply:
Is it ok for a struct to inherit from a class? Seems weird

Reply:
Yep.
Literally the only difference between a struct and a class is the default access permissions (public / protected / private)

struct = everything public by default
class = everything private by default

No other differences.